src/components/TopMenu.js

```diff
@@ -9,9 +9,9 @@ export const TopMenu = ({ topMenu, activeDocMeta }) => {
   const renderedMenuLinks = topMenu
     ? topMenu.menu_links.map((menuLink, index) => (
         <li key={`top-nav-${index}`}>
-          <Link id={menuLink.link.id} to={menuLink.link.url}>
-            {RichText.asText(menuLink.label.richText)}
-          </Link>
+          <PrismicLink href={menuLink.link.url}>
+            <PrismicText field={menuLink.label.richText} />
```

Comment:
Does `asText` and `PrismicText` work in the same way, or what is the difference?

Reply:
That's right; `asText()` and `<PrismicText>` output the same text. Only the syntax is different.

- `asText()` can be used in normal JavaScript and in JSX, but requires `{...}` to work in JSX.
- `<PrismicText>` can be used in JSX like any other JSX element.

A little more explanation is given here: https://prismic.io/docs/technologies/prismic-react-v2-migration-guide#replace-richtext.astext-with-prismictext-or-astext